### DISCUSSION TITLE
Connected to http://xxx.xxx.xxx.xxx:8400 Meldung

### DISCUSSION THREAD

**Discussion by holofury** [Zefau/ioBroker.jarvis#2353] [Q&A]
Moin,

Jarvis läuft bei mir über den Fully Browser auf einem HUAWEI MatePad T 10.

Bei jedem Aufwecken des Tablets erscheint die Meldung "Connected to http://xxx.xxx.xxx.xxx:8400" als Banner unten für ein paar Sekunden.

Ich bin mir sicher, dass man das irgendwo ausschalten kann, finde es aber nicht...
Kann mir da jemand helfen?

**Comment by mcuiobroker** [comment]
Kann man nicht abschalten.

**Comment by holofury** [comment]
Kommt die Meldung denn von Jarvis oder vom FullyBrowser?

**Comment by Zefau** [comment]
Die Meldung kommt von jarvis und sollte nur erscheinen, wenn es zuvor Verbindungsprobleme gab.

**Reply by holofury** [reply]
Die Meldung kommt bei mir, wie geschrieben, bei dem Aufwecken des Tablets.
Auf dem Notebook erscheint nichts.

Log im ioBroker zeigt nur die An- und Anmeldung des Tablets. 

Hast du einen Tipp, wie ich die Meldung los werde?

**Comment by holofury** [comment]
Irgendein Tipp?

**Comment by mcuiobroker** [comment]
Wenn das Tablet sich Schlafen legt, wird die Verbindung abgebrochen. Dies wird beim Aufwecken dann angezeigt und weist daraufhin, dass es eine Unterbrechung der Verbindung gegeben hat. Das ist normal und nicht problematisch. Wenn die Meldung gar nicht kommen soll, müsste man eine Option einbauen, um diese explizit auszuschalten.
-> Feature Request.

Alternativ nur die Helligkeit ändern, somit bleibt die Verbindung erhalten.
